lib/rspec/core/formatters/minimal_formatter.rb

```diff
@@ -0,0 +1,23 @@
+RSpec::Support.require_rspec_core "formatters/base_formatter"
+
+module RSpec
+  module Core
+    module Formatters
+      # @private
+      class MinimalFormatter < BaseFormatter
```

Comment:
"Minimal" is a subjective term -- I can imagine many different minimal formatter that output different things.  Here it seems like this formatter is focused on printing a list of failures.  WDYT about calling this `FailureListFormatter`?  Seems like a more accurate name.

Reply:
Hehe, [naming things is hard](https://www.martinfowler.com/bliki/TwoHardThings.html)…

The formatter was initially called [CFormatter](https://github.com/rspec/rspec-core/commit/2769fb3dc66915d0851b7a28b7212347f5f64023) — which was probably an even poorer choice — because it's output look like what a compiler produce…

Reply:
I'm happy to change the name, `FailureListFormatter` seems appropriate.

Reply:
See https://github.com/rspec/rspec-core/pull/2624